From 22f44d68b16f48a9a43f300bc1d1612aa91a45f5 Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Thu, 11 Jun 2020 00:50:00 -0700
Subject: [PATCH 1172/1239] Revert "board: octeontx3: check for chip type in
 fuse register"

This reverts commit b141873d9b541c343aa193df57e299afe3482a7d.

Reason for revert: Incorrect platform name

Change-Id: Ib44effc9372ba64f0b6add5a36860d11cbdb9df7
Signed-off-by: Suneel Garapati <sgarapati@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/boot/u-boot/+/29829
Reviewed-by: Suneel Garapati <sgarapati@caviumnetworks.com>
Tested-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 board/Marvell/octeontx3/soc-utils.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/board/Marvell/octeontx3/soc-utils.c b/board/Marvell/octeontx3/soc-utils.c
index 6b81cae07c..abf355bd6f 100644
--- a/board/Marvell/octeontx3/soc-utils.c
+++ b/board/Marvell/octeontx3/soc-utils.c
@@ -42,12 +42,9 @@ u8 read_partvar(void)
 	return ((read_midr() >> 20) & 0xF);
 }
 
-#define FUS_CACHE0_REG 0x87e003001000
 u8 read_partnum(void)
 {
-	u64 fus_cache0 = readq(FUS_CACHE0_REG);
-
-	return (u8)(fus_cache0 & 0xff);
+	return ((read_midr() >> 4) & 0xFF);
 }
 
 const char *read_board_name(void)
-- 
2.29.0

